### DISCUSSION TITLE
Is it possible to watch a FamilyAsyncNotifier without adding the parameter for the family?

### DISCUSSION THREAD

**Discussion by aidanmack** [rrousselGit/riverpod#2755] [Q&A]
So I start watching the provider in one widget with

ref.watch(fullStationDetailsProvider(keys))

But then I want another widget to also observe the same fullstationsDetailsProvider that I initialised in the previous widget.

So this doesnt work...
ref.watch(fullStationDetailsProvider())
...as it expects keys.

Anyone tell me what the solution is here?

Thanks

**Comment by catalunha** [comment]
I already asked myself this question. And I thought I could start at a location and then use it without the parameter. It would be interesting.
I still have a lot of doubts about the Riverpod, but I'm not going to abandon it. Remi and his team are fantastic.
But let's see the answer of the more experienced.

**Comment by rrousselGit** [comment]
No, it isn't possible. It might be in the future through scoping + lints, but not now.

**Comment by catalunha** [comment]
Thanks
